book/src/intro.md

```diff
@@ -15,46 +18,53 @@ _Documentation for the Kona project._
 
 ## Introduction
 
-Kona is _the_ monorepo for the Rust Optimism <a href="https://specs.optimism.io/">OP Stack</a>.
+Kona is _the_ monorepo for the Rust <a href="https://specs.optimism.io/">OP Stack</a>.
 
-Kona provides a suite of libraries and build pipelines for developing verifiable Rust programs targeting
-{{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
+Kona provides a rich suite of libraries and build pipelines. Proof libraries are available for
+developing verifiable Rust programs targeting {{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
```

Comment:
This is a bit legacy from back before we generalized the proof crates a bit more; The proof libraries are more generic, and allow for building verifiable programs that run the OP Stack (base / extended) state transition. There are also platform-specific crates for the FPVM, which are more core for performing IO, etc.